README.md

```diff
@@ -2,87 +2,116 @@
+* Editor kept in sync with the resource on cluster by pushing & pulling
+
+## Features
+
+### Browse Kubernetes & OpenShift resources
+This plugin allows the user to view the Kubernetes and OpenShift Cluster resources in the Application Explorer Tree and browser through them.
+The explorer tree is dynamically updated in sync with the Kubernetes resources on the cluster so that the tree always display the current state of the resources.
 
 #### Current Context
 The tree displays resources that exist on the cluster that the current context points to. 
-The user can switch the current context and cluster by choosing any context that's listed and picking "Set as Current Cluster" in the context menu.
-OpenShift clusters are marked as such with an OpenShift icon instead of a Kubernetes icon.
+The user can switch the current context by choosing any context that exists and selecting `Set as Current Cluster` action from the context menu.
+OpenShift clusters are shown with OpenShift icon and thus differentiated from Kubernetes clusters.
  
 #### Current Namespace/Project
-The tree displays resources that exist within the current namespace as specified in the kube config or that are not bound to a namespace/project.
-The user may switch the current namespace with the context menu item "Use Namespace". 
-In OpenShift clusters the tree also lists projects which behave in an equivalent manner. 
+Kubernetes Resources that exist within the current Namespace/Project (as specified in kubeconfig file) are displayed in the tree view 
+Of course resources that live outside the scope of namespace/project are displayed, too.
+Users can switch the current Namespace/Project using the context menu action `Use Namespace`
+In OpenShift clusters the tree also lists projects where the tooling behaves in an equivalent manner. 
 
 #### Resource Categories
-Resources are organized in different categories.
+Resources are grouped in different categories such as Namespaces, Nodes, Workloads, Network, Storage, Configuration and Custom Resources.
 
 ![img.png](images/categories.png)
 
 #### Pods
-Pods are marked as running if their icon has a green dot. 
-A red dot, on the contrary, indicates that a pod is either pending, succeeded (terminated), failed or is in an unknown state.
-Pods also unveil their IP address and the number of running out of the total number of containers.
-
+Pods are shown with their running state. When running their icon holds a green dot. 
+When red, the dot indicates that a pod is either pending, succeeded (terminated), failed or is in an unknown state.
+Pods also unveil their IP address and the number of running containers out of the total number of containers.
+The pod in the container is identified with color based on the state of the pod.
+- A green dot indicates that the pod is currently running.
+- A red dot indicates that the pod is in Pending, Succeeded (Terminated), Failed or in Unknown State.
 ![img.png](images/pod.png)
```

Comment:
@adietish Here also the lines are repeated. We need to remove line no 51,52. And move 54-56 above line 53.

Reply:
@mohitsuman is this what your instructions result into? Please confirm:
```
The pod in the container is identified with color based on the state of the pod.
- A green dot indicates that the pod is currently running.
- A red dot indicates that the pod is in Pending, Succeeded (Terminated), Failed or in Unknown State.  

![img.png](images/pod.png)

Pods also unveil their IP address and the number of running containers out of the total number of containers.
```

Reply:
Yes this is the intended change.